### DISCUSSION TITLE
[projects beta] Add a 'collapse all groups' action

### DISCUSSION THREAD

**Discussion by Flamenco** [community/community#10080] [Projects and Issues]
Perhaps put it right above "**Rename View**" in the tab settings"

<img width="411" alt="Screen Shot 2022-01-13 at 11 03 29 AM" src="https://user-images.githubusercontent.com/819940/149365644-c73fd109-6d5a-418d-a768-4b0225630268.png">

**Comment by mikermcneil** [comment]
+1
Would be awesome if it could stay saved as part of the view too.

<img width="888" alt="image" src="https://user-images.githubusercontent.com/618009/159604151-8ad8943c-aeef-4c95-b862-30f062f01b0c.png">


(Maybe?) ~~relates to https://github.com/github/feedback/discussions/7932~~

**Comment by mikermcneil** [comment]
Definitely relates to https://github.com/github/feedback/discussions/6579 and https://github.com/github/feedback/discussions/8566

**Comment by mxkpp** [comment]
IMO this Issue is better than other related ones due to it asking for one feature-add instead of a laundry list.
